hardly/handlers/distgit.py

```diff
@@ -130,6 +131,19 @@ def packit(self) -> PackitAPI:
             )
         return self._packit
 
+    def sync_release(self):
+        return self.packit.sync_release(
+            dist_git_branch=self.target_repo_branch,
+            version=self.packit.up.get_specfile_version(),
+            add_new_sources=False,
+            title=self.mr_title,
+            description=f"{fix_bz_refs(self.mr_description)}\n\n---\n{dg_mr_info}",
```

Comment:
`dg_mr_info` is not defined in this scope.

Reply:
Right. I moved all the dg_mr_info references in the sync_release function